control_panel_api/views.py

```diff
@@ -45,6 +45,14 @@ class AppViewSet(viewsets.ModelViewSet):
     filter_backends = (AppFilter,)
     permission_classes = (AppPermissions,)
 
+    def perform_create(self, serializer):
+        app = serializer.save()
+        services.app_create(app.slug)
+
+    def perform_destroy(self, instance):
+        instance.delete()
+        services.app_delete(instance.slug)
```

Comment:
Is the instance and it's slug still valid here after deletion?

Reply:
I tried to inspect using `pdb` and the app's slug was there after the `delete` (but predictably not the `id` :D) - Also, tests pass so looks like get the app slug like this is valid.

Do you have any strong preference to get it into a variable before the `delete` and want me to change it?